src/lib.rs

```diff
@@ -75,11 +75,16 @@ pub async fn merge_main(opt: Opt) -> Result<(), anyhow::Error> {
     let pusher_config = MergeConfig::parse_file(&opt.merger_config_path)?;
 
     let external_execution_config = match &opt.external_config_path {
-        Some(path) => {
-            let json_str = fs::read_to_string(&path)?;
-            Some(serde_json::from_str(&json_str)?)
+        Some(paths) => {
+            let mut configs = Vec::new();
+            for p in paths {
+                let json_str = fs::read_to_string(&p)?;
+                let config = serde_json::from_str(&json_str)?;
```

Comment:
can we use https://docs.rs/serde_json/latest/serde_json/fn.from_reader.html to read the file directly rather than read string then string to json?